src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1257,7 +1257,16 @@ public void onAddTable() {
 
 	private void postCheckCreateTableOrView(TableType type) {
 		getCreateTableViewWizard().configure(entityBundle.getEntity().getId(), type);
-		getCreateTableViewWizard().showModal(entityUpdatedWizardCallback);
+		getCreateTableViewWizard().showModal(new WizardCallback() {
+			@Override
+			public void onFinished() {
+				// Intentionally empty
+				// The modal will redirect us to the appropriate Place, which will trigger a full page update
```

Comment:
The previous value of `onFinished` invoked `fireEntityUpdatedEvent`. The wizard calls `PlaceChanger.goTo` right before invoking `onFinished`, so we triggered two updates. I still don't understand why calling `fireEntityUpdatedEvent` immediately after changing places to the new entity would cause all of the React components in EntityPageTop to break.

Reply:
Does not make sense to me either, but since the Placechanger.goTo will trigger a page update we don't need the EntityUpdatedEvent on the gwt event bus, so I think this is a safe change to make.